Tests/ViewInspectorTests/SwiftUI/CustomViewModifierTests.swift

```diff
@@ -177,6 +177,144 @@ final class ModifiedContentTests: XCTestCase {
         XCTAssertEqual(sut.content.medium.environmentModifiers.count, 1)
         XCTAssertEqual(sut.content.medium.environmentObjects.count, 0)
     }
+
+    func testMultipleTransitiveModifiers() throws {
+        let view = Text("str")
+            .allowsHitTesting(true)
+            .allowsHitTesting(true)
+            .allowsHitTesting(true)
+            .allowsHitTesting(true)
+            .allowsHitTesting(true)
+            .allowsHitTesting(true)
+            .allowsHitTesting(true)
+            .allowsHitTesting(true)
+            .allowsHitTesting(true)
+            .allowsHitTesting(true)
+            .allowsHitTesting(true)
+            .allowsHitTesting(true)
+            .allowsHitTesting(true)
+            .allowsHitTesting(true)
+            .allowsHitTesting(true)
+
+        let sut = try view.inspect()
+        XCTAssertEqual(sut.content.medium.transitiveViewModifiers.count, 15)
+        XCTAssertEqual(sut.content.medium.viewModifiers.count, 0)
+    }
+
+    func testMultipleCustomTransitiveModifiers() throws {
+        let view = Text("str")
+            .modifier(AllowHitTestingTransitiveModifier())
+            .modifier(AllowHitTestingTransitiveModifier())
+            .modifier(AllowHitTestingTransitiveModifier())
+            .modifier(AllowHitTestingTransitiveModifier())
+            .modifier(AllowHitTestingTransitiveModifier())
+            .modifier(AllowHitTestingTransitiveModifier())
+            .modifier(AllowHitTestingTransitiveModifier())
+            .modifier(AllowHitTestingTransitiveModifier())
+            .modifier(AllowHitTestingTransitiveModifier())
+            .modifier(AllowHitTestingTransitiveModifier())
+            .modifier(AllowHitTestingTransitiveModifier())
+            .modifier(AllowHitTestingTransitiveModifier())
+            .modifier(AllowHitTestingTransitiveModifier())
+            .modifier(AllowHitTestingTransitiveModifier())
+            .modifier(AllowHitTestingTransitiveModifier())
+
+        let sut = try view.inspect()
+        XCTAssertEqual(sut.content.medium.transitiveViewModifiers.count, 15)
```

Comment:
This test fails this assertion in the state of f52f300f10df9a5980dcf839f0048a9dcccf6e72 (the first commit of this branch).

The `transitiveViewModifiers.count` returns **32767** items.

<img width="1102" alt="Screenshot 2023-12-13 at 10 07 05 PM" src="https://github.com/nalexn/ViewInspector/assets/2278022/f179a112-f986-4f43-b1fc-beeb9572a6dd">